osidb/api_views.py

```diff
@@ -345,6 +354,20 @@ def create(self, request, *args, **kwargs):
         return response
 
 
+@include_meta_attr_extend_schema_view
+@include_exclude_fields_extend_schema_view
+@extend_schema_view(
+    create=extend_schema(
+        request=FlawAcknowledgmentPostSerializer,
+    ),
+)
+class FlawAcknowledgmentView(ModelViewSet):
+    queryset = FlawAcknowledgment.objects.all()
```

Comment:
This queryset won't take into account the `flaw_id` path parameter and will include all acknowledgements. Similar handling as for Flaw Comments or Flaw References should be done.

Reply:
Fixed in commit 3c2d2dd147380fc8924fac4b9d539f85f86f36d4